### PR TITLE
Fix UI in Property component & Removed reexports from deskpro-ui

### PR DESCRIPTION
**Pull request by zpawn**
Stories

- https://app.shortcut.com/deskpro/story/109670/app-sdk-fix-property-children
- https://app.shortcut.com/deskpro/story/100144/app-sdk-update-dependencies

### CODE REVIEW

File: src/ui/components/TwoButtonGroup/TwoButtonGroup.stories.tsx [hunk 1]
```diff
@@ -33,6 +34,6 @@ Default.args = {
     selected: "one",
     oneLabel: "Find",
     twoLabel: "Create",
-    oneIcon: true,
-    twoIcon: true,
+    oneIcon: true as never,
```
Comment (amDavid01) [inline src/ui/components/TwoButtonGroup/TwoButtonGroup.stories.tsx:37]: why is never being used here? never is for code that should be unreachable

Comment (zpawn) [reply src/ui/components/TwoButtonGroup/TwoButtonGroup.stories.tsx:37]: to silence TS warning

Comment (amDavid01) [reply src/ui/components/TwoButtonGroup/TwoButtonGroup.stories.tsx:37]: arent there better types that you can use like unknown?

Comment (zpawn) [reply src/ui/components/TwoButtonGroup/TwoButtonGroup.stories.tsx:37]: there is an error on `unknown` type

Comment (zpawn) [reply src/ui/components/TwoButtonGroup/TwoButtonGroup.stories.tsx:37]: In your opinion, what is better to set?

Comment (amDavid01) [reply src/ui/components/TwoButtonGroup/TwoButtonGroup.stories.tsx:37]: Id need to test but i dont really have time, if you dont wanna see that pass

**Review by amDavid01 — APPROVED** [review]

**Review by amDavid01 — APPROVED** [review]

**Pull request merged**